src/reducers/books.js

```diff
@@ -0,0 +1,35 @@
+const initialState = {
+  books: [{
+    id: Math.random,
+    title: 'First Book',
+    cat: 'Action',
+  },
+  {
+    id: Math.random,
+    title: 'Second Book',
+    cat: 'Biography',
+  },
+  {
+    id: Math.random,
+    title: 'Third Book',
+    cat: 'History',
```

Comment:
### [Optional]
Kindly be aware that you can use uuidv4 to automatically generate ids. read more 👉 [npm uuid](https://www.npmjs.com/package/uuidv4)

Example:
```
import uuid from 'uuidv4'
id: uuid(): {title: 'First Book', category: 'Actioin'}
```